src/main/java/programminglife/utility/Alerts.java

```diff
@@ -0,0 +1,158 @@
+package programminglife.utility;
+
+import javafx.application.Platform;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
+
+import java.util.Optional;
+
+/**
+ * A class which contains all the possible alerts we can give to the user.
+ */
+public final class Alerts {
+    private static final double ABOUT_MIN_WIDTH = 500;
+    private static final double INSTRUCTIONS_MIN_WIDTH = 800;
+
+    private static Alerts alerts;
+
+    /**
+     * Constructor for the Alert.
+     */
+    private Alerts() { }
+
+    /**
+     * return an instance of the alert class.
+     * @return Alerts.
+     */
+    public static synchronized Alerts getInstance() {
+        if (alerts == null) {
+            alerts = new Alerts();
+        }
+        return alerts;
+    }
+
+    /**
+     * Alert method used if a file can't be opened.
+     */
+    public void cantOpenAlert() {
```

Comment:
1. Shouldn't these methods be `static`? `Alerts.randomAlert()` is better than
```java
private Alerts alerts = Alerts.getInstance();
this.alerts.randomAlert();
```
in my opinion.
2. I think this class would be a lot more versatile if it had
```java
public Alert error(String msg) {
  return new Alert(AlertType.ERROR, msg, ButtonType.CLOSE);
}
```
and similar methods for `AlertType.WARNING` etc. This prevents more duplicate code. Also, this allows the caller of the method to operate on the returned `Alert`, e.g. use `Alert.showAndWait()` if necessary.